tox.ini

```diff
@@ -1,24 +1,21 @@
 [tox]
 envlist =
     quality
-    py35
-    py36
-    py37
     py38
     py39
     clear-coverage
     compute-coverage
 skip_missing_interpreters = True
 
-# Mark quality as a part of python3.6 build
+# Mark quality as a part of python3.9 build
 [travis]
 python =
-    3.6: py36, quality
+    3.9: py39, quality
```

Comment:
This section was Travis specific. We do not use it anymore, so can be safely removed.